### PR TITLE
Blockquote

### PR DESCRIPTION
**Pull request by CaptainFalcon92**

### CODE REVIEW

File: src/MarkdownBuilder.php [hunk 1]
```diff
@@ -76,7 +76,7 @@ public function h3($header)
      * @param string $text
      * @return $this
      */
-    public function blockqoute($text)
+    public function blockquote($text)
```
Comment (DavidBadura) [inline src/MarkdownBuilder.php]: can you add a bc method?

```
/**
 * @deprecated
 */
public function blockqoute($text) {
    return $this->blockquote($text);
}
```

so i can it release as a patch level update.

Comment (CaptainFalcon92) [reply src/MarkdownBuilder.php]: Hello.
I was just passing by - hope this helps.
Your library is a usefull companion for data fixtures. Thanks.

Comment (DavidBadura) [reply src/MarkdownBuilder.php]: nice, thank you!

**Comment by DavidBadura** [comment]
just release 1.0.4 https://github.com/DavidBadura/markdown-builder/releases/tag/1.0.4

**Pull request merged**